README.md

```diff
@@ -1 +1,39 @@
-# Training-mobile-menu
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Training Mobile Menu
+
+In this project, I trained how to add a mobile menu
+
+<br>
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Author
+
+👤 **Abenezer Tilahun**
+
+- GitHub: [@Abenezer-Tilahun](https://github.com/Abenezer-Tilahun)
+- Twitter: [@AbenezerTilah11](https://twitter.com/AbenezerTilah11)
+- [LinkedIn](https://www.linkedin.com/in/abenezer-tilahun/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- A special thank for @fernandorpm for this amazing [README template](https://github.com/microverseinc/readme-template)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] On your readme, you have the section "license" according to the [Microverse readme template](https://github.com/microverseinc/readme-template).  Kindly, make sure that The link featured in that section should point to a file at the root of your project (the license file).   Just copy it from the mentioned template:

![image](https://user-images.githubusercontent.com/57421823/168004669-c3026da8-8102-4a87-bb49-f7a5a9a6abbf.png)

That way, you will have a professional repo, and your project will be well documented 💪